spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -201,8 +201,8 @@ def build_trains
 
       # The numbers of all build trains that were uploaded
       # @return [Array] An array of train version numbers
-      def all_build_train_numbers
-        client.all_build_trains(app_id: self.apple_id).fetch("trains").collect do |current|
+      def all_build_train_numbers(platform: nil)
```

Comment:
I understand that both gems will need to be released, and the dependency will need to be bumped. But why does it require two PRs?